python/src/BetaMuSigma_doc.i.in

```diff
@@ -39,3 +42,18 @@ Convert parameters into the native parameters :math:`(\alpha, \beta, a, b)`:
 
 >>> print(parameters.evaluate())
 [2,3,-1,2]"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::BetaMuSigma::isElliptical
```

Comment:
there is no such method

Reply:
In the file BetaMuSigma.cxx, there is a method isElliptical that answers 'Yes' when  b = 2.0 * mu - a....

Reply:
oh sorry